packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="success">
+    <template v-slot:header>
+      <div class="context__header">{{ appmapStats.numAppMaps }} AppMaps available</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__table mb-0">
+        <div class="context__table__header">Navie's context:</div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Runtime traces (AppMap recordings)</div>
+        </div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Source code snippets</div>
+        </div>
+      </div>
+    </template>
+  </v-status-bar>
+
+  <v-status-bar v-else level="error">
+    <template v-slot:header>
+      <div class="context__header">No AppMaps available</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__sub-header">
+        <div class="context__table">
+          <div class="context__table__header">Navie's context:</div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Runtime traces (AppMap recordings)</div>
+          </div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Source code snippets</div>
+          </div>
+        </div>
+        <p class="mt-0">
+          You can still ask Navie questions, but Navie does not have access to your project's data.
+        </p>
+        <template v-if="!appmapYmlPresent">
+          <p class="mb-0 mt-0" data-cy="status-no-config">
```

Comment:
Absence of appmap.yml doesn't mean that the user hasn't followed the instructions. For example, with Node.js, all they have to do is `npx appmap-node <cmd>`. Sending them back to the instructions won't fix the problem of missing appmap.yml. What they need to do is run their app using a recording method. We've made sure that all our recording methods will generate and use a default appmap.yml. So, I don't think "missing appmap.yml" is really a state that we can give specific treatment to.

Reply:
In other words, I think we can drop this section.